### DISCUSSION TITLE
Two new features that can improve usability greatly.

### DISCUSSION THREAD

**Discussion by crocket** [qutebrowser/qutebrowser#6687] [Ideas] [answered]
* Configurable job scheduler for updating adblock lists and doing other maintenance tasks automatically
* ABP-style cosmetic filter that eliminates ugly HTML elements.

These two are blocking total migration to qutebrowser from firefox.

Qutebrowser is the only web browser that allows full configuration in plain text file.
Other web browsers suck because they don't support plain text configuration.

**Comment by The-Compiler** [comment, marked as answer]
See:

- #737
- #6480

**Reply by crocket** [reply]
I forgot to mention the related issues. The idea is that you get 51% of results by focusing on 1% of all issues.
That's roughly 9 issues.
